### PR TITLE
refactor(*): feedback review from @Mouvedia

### PR DESCRIPTION
**Pull request by topheman**
Comments can be found at https://github.com/Mouvedia/snake-pipe-rust/pull/1

### CODE REVIEW

File: node-helpers/dev-server-sse/index.ts [hunk 1]
```diff
@@ -30,17 +41,13 @@ if (path.isAbsolute(filePathOfGameRecording)) {
...
-}
-
 async function* infiniteAsyncGeneratorFromArrayString(input: Array<string>, delay = 120) {
   let currentIndex = 0;
   yield input[currentIndex];
   while (true) {
     currentIndex++;
     if (input[currentIndex]?.trim()) {
-      await timeout(120);
+      await setTimeout(delay);
```
Comment (Mouvedia) [inline node-helpers/dev-server-sse/index.ts:50]: J'avais raté ça.

File: node-helpers/dev-server-sse/index.ts [hunk 2]
```diff
@@ -1,21 +1,32 @@
...
 import { makeServer } from './server.js'
 
-
+/**
+ * `__dirname` doesn't exist in esm, you need to create it.
+ *
+ * We could improve the code bellow with just `import.meta.dirname` - it needs at least node 20.11.
+ * For the moment keeping retrocompatibility with all node 20 versions.
+ *
+ * Source: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-module.md
```
Comment (Mouvedia) [inline node-helpers/dev-server-sse/index.ts:17]: meilleur source : https://nodejs.org/docs/v20.11.0/api/esm.html#importmetadirname

File: node-helpers/dev-server-sse/index.ts (hunk 1 above)
Comment (topheman) [reply node-helpers/dev-server-sse/index.ts:50]: `timersPromises.setTimeout([delay[, value[, options]]])` https://nodejs.org/api/timers.html#timerspromisessettimeoutdelay-value-options

`value` is optional, eslint doesn't identify it's not the regular `setTimeout` ?

Comment (Mouvedia) [reply node-helpers/dev-server-sse/index.ts:50]: I meant you wouldn't have missed that you were not using `delay` using this rule.

Comment (topheman) [reply node-helpers/dev-server-sse/index.ts:50]: ok, for the moment, I haven't setup any eslint for this part.

**Pull request merged**